python/tvm/relay/op/vision/nms.py

```diff
@@ -0,0 +1,36 @@
+"""Non-maximum suppression operations."""
+from __future__ import absolute_import as _abs
+from . import _make
+
+def nms(data,
+        valid_count,
+        nms_threshold=0.5,
+        force_suppress=False,
+        nms_topk=-1):
+    """Generate prior(anchor) boxes from data, sizes and ratios.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        3-D tensor with shape [batch_size, num_anchors, 6].
+        The last dimension should be in format of
+        [class_id, score, box_left, box_top, box_right, box_bottom].
+
+    valid_count : relay.Expr
+        Number of valid anchor boxes.
```

Comment:
It might be more understandable if information which `valid_count` is `1-D tensor` is specified as with the parameter description in topi's nms.

```
1-D tensor for valid number of boxes.
```